website/content/docs/enterprise/license/faqs.mdx

```diff
@@ -0,0 +1,102 @@
+
+Also, when customers deploy new clusters to a 1.8(+) release, a valid license must exist to upgrade successfully. This valid license must be on-disk (auto-loaded).
+
+## Q: What scenarios should a customer plan for due to these license changes?
+
+- **New Cluster Deployment**: When a customer deploys new clusters to a 1.8(+) release, a valid license must exist to successfully deploy. This valid license must be on-disk (auto-loaded).
+
+- **Eventual Migration**: Since in-storage licenses will not be supported forever, migrating to auto-loaded licenses will be required. Please see the below question: [Is there a limited time for support of licenses that are in storage?](#q-is-there-a-limited-time-for-support-of-licenses-that-are-in-storage).
+
+## Q: What is the migration path for customers who want to migrate from their existing license-as-applied-via-the-CLI flow to the license on disk flow?
+
+If a Vault 1.7 cluster is upgraded to Vault 1.8(+) and still has a stored license, the operator must migrate using an auto-loaded license.
+
+1. Use the new command `vault license get -signed` to retrieve the license from the storage in their running cluster.
+2. Put the license on disk, configure auto-loading (new license_path config option, OR new environment variable `VAULT_LICENSE_PATH` OR new environment variable `VAULT_LICENSE` containing the actual license itself).
+3. Restart the Vault node.
+4. A warning will explain that the license is in storage and using auto-loading.
+5. Use the new command `delete /sys/license` to fully transition to auto-loading.
+
+## Q: What is the migration path for customers who wish to migrate from their existing perpetually licensed binaries to the license on disk flow (auto-loading)?
+
+A Vault customer who is currently using S3 unlocked binaries, upgrading their clusters to Vault 1.8(+), and wishes to move to auto-loading, will need a valid enterprise binary version 1.8(+) and a valid license. The steps are the same as in the above question: [What is the migration path for customers who want to migrate from their existing license-as-applied-via-the-CLI flow to the license on disk flow?](#q-what-is-the-migration-path-for-customers-who-want-to-migrate-from-their-existing-license-as-applied-via-the-cli-flow-to-the-license-on-disk-flow) starting at Step 2.
+
+## Q: What is the migration path for customers who want to migrate from their existing perpetually licensed binaries to the current license on storage flow?
+
+A Vault customer currently using S3 unlocked binaries is upgrading their clusters to a pre-Vault 1.8 version, the customer must retrieve the corresponding enterprise binary version <1.8 and a valid license to load the license in storage (/sys/license). Please see the below question: [Is there a limited time for support of licenses that are in storage?](#q-is-there-a-limited-time-for-support-of-licenses-that-are-in-storage).
```

Comment:
You already said, "pre-Vault 1.8 version", so I don't think you need to say "version <1.8". 

```suggestion
A Vault customer currently using S3 unlocked binaries is upgrading their clusters to a pre-Vault 1.8 version, the customer must retrieve the corresponding enterprise binary and a valid license to load the license in storage (`/sys/license`). Please see the below question: [Is there a limited time for support of licenses that are in storage?](#q-is-there-a-limited-time-for-support-of-licenses-that-are-in-storage).
```